app/controllers/verify_controller.rb

```diff
@@ -15,12 +15,16 @@ def show
   private
 
   def app_data
+    cipher = Encryption::AesCipher.encryption_cipher
+    user_session[:idv_api_store_key] ||= Base64.strict_encode64(cipher.random_key)
```

Comment:
I'm not sure how expensive allocating the cipher is, but we could make this a bit lazier if we wanted to by inlining it

```suggestion
    user_session[:idv_api_store_key] ||= Base64.strict_encode64(
      Encryption::AesCipher.encryption_cipher.random_key,
    )
```

Reply:
Good call! Updated in 5384a34ff9e3be226fe0cae01ae81581dd5d56b3.